src/backend/PhotoAPI/DataAccess/Implementations/PhotoBlobStorage.cs

```diff
@@ -107,5 +112,24 @@ public Task<bool> DeleteFileIfExistsAsync(string blobName)
 			CloudBlockBlob blob = _cloudBlobContainerPhotos.GetBlockBlobReference(blobName);
 			return blob.DeleteIfExistsAsync();
 		}
+
+		public async Task<IEnumerable<FileItem>> DownloadAsync(IEnumerable<string> fullBlobNames)
+		{
+			IEnumerable<Task<ICloudBlob>> cloudBlobsTasks = fullBlobNames
+				.Select(Path.GetFileName)
+				.Select(blobName => _cloudBlobContainerPhotos.GetBlobReferenceFromServerAsync(blobName));
+
+			IEnumerable<ICloudBlob> cloudBlobs = await Task.WhenAll(cloudBlobsTasks);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=5223686)